google/cloud/bigtable/internal/async_poll_op_test.cc

```diff
@@ -61,6 +61,8 @@ class DummyOperation {
 
   DummyOperation(std::shared_ptr<DummyOperationImpl> impl) : impl_(impl) {}
 
+  virtual ~DummyOperation() = default;
```

Comment:
It doesn't appear to me that DummyOperation needs to have virtual behavior in any of these.  That is handled by DummyOperationImpl.